### PR TITLE
internetarchive: Update README

### PR DESCRIPTION
**Pull request by mitra42**
# Description of changes proposed in this pull request.
Update internetarchive README.md

### Mention a team member for further information or comment using @ name
@holta

### CODE REVIEW

File: roles/internetarchive/README.md [hunk 1]
```diff
@@ -1,71 +1,284 @@
-# Internet Archive Offline / Universal Library / Decentralized Web README
+# Offline Internet Archive
```
Comment (holta) [inline roles/internetarchive/README.md]:
```suggestion
# Offline Internet Archive README
```

Comment (holta) [reply roles/internetarchive/README.md]: By tradition, IIAB apps/services have 'README' in their doc title as such.

**Comment by holta** [comment]
@mitra42 this could use a really quick cleanup of stray punctuation (dangling commas etc, consistent spacing between paragraphs) if you can give it a fast once-over ideally?

**Comment by mitra42** [comment]
OK - done a minor cleanup, this doc is cut and pasted from a number of other docs because of the difference of the IIAB platform, which means that any edits made here, and not on dweb-mirror/[README.md](https://github.com/dweb-mirror/README.md) and [USING.md](https://github.com/dweb-mirror/) are likely to be lost next time there is an update.

**Pull request merged**